src/components/Supplier/AddSupplier.tsx

```diff
@@ -1,22 +1,34 @@
 import { VStack, StackDivider, Box, Flex } from '@chakra-ui/react';
 import AddSupplierTitle from 'components/Supplier/AddSupplierTitle';
-import AddSupplierButtons from 'components/Supplier/AddSupplierButtons';
 import AddSupplierData from 'components/Supplier/AppSupplierData/AddSupplierData';
+import styled from 'styled-components';
 
-const Form = () => (
-  <Flex w="96rem" p="1em">
-    <Box w="22em" h="30em" bg="gray.200" />
-    <Box w="1em" h="1em" bg="white" />
-    <VStack w="72em" divider={<StackDivider borderColor="gray.200" />} spacing={3} align="stretch">
-      <Box h="2em">
-        <AddSupplierTitle />
-      </Box>
-      <Box h="26em" w="70em">
-        <AddSupplierData />
-      </Box>
-      <AddSupplierButtons />
-    </VStack>
-  </Flex>
-);
+const Form = () => {
+  const StyledDiv = styled.div`
```

Comment:
I'm considering dropping Styled Components altogether. I've familiarized myself a bit with Chakra and it uses [Emotion](https://emotion.sh/docs/introduction) under the hood, it's a different "CSS in JS" library, so it probably doesn't make much sense to use Styled Components and Emotion together.

Referring directly to the code - why are you writing a special Styled Component just to set the width and padding? Note that below you are using components from Chakra and applying the props that Chakra provides to achieve the same effect. 

The purpose of going into a UI library is to follow conventions, design system, etc., so creating this type of Styled Component makes absolutely no sense. As for the responsiveness issue, please [read the Chakra documentation](https://chakra-ui.com/docs/features/responsive-styles), [here you can find examples](https://chakra-ui.com/docs/features/responsive-styles#more-examples) of how to apply responsiveness.